package.json

```diff
@@ -2,56 +2,55 @@
   "name": "ember-cli-sass",
   "version": "7.0.0",
   "description": "Use node-sass to preprocess your ember-cli app's files, with support for sourceMaps and include paths",
+  "keywords": [
+    "ember-addon",
+    "sass"
+  ],
+  "license": "MIT",
+  "author": "@aexmachina",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/aexmachina/ember-cli-sass.git"
+  },
   "scripts": {
     "build": "ember build",
     "start": "ember server",
     "test": "ember try:each"
   },
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/aexmachina/ember-cli-sass.git"
-  },
-  "engines": {
-    "node": ">= 0.10.0"
+  "dependencies": {
+    "broccoli-funnel": "^1.0.0",
+    "broccoli-merge-trees": "^1.1.0",
+    "broccoli-sass-source-maps": "^2.0.0",
+    "ember-cli-babel": "^5.1.7",
```

Comment:
it seems like this dependency got added by mistake. I don't think it is actually needed 🤔

Reply:
Good catch @Turbo87